www.foia.gov/_includes/header.html

```diff
@@ -1,78 +1,73 @@
 <!-- Start Header -->
 <script async type="text/javascript" id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOJ"></script>
 <script type="text/javascript" src="{{ site.baseurl }}/scripts/breakout.js"></script>
-{% if page.use-uswds %}
-  <a class="usa-skipnav" href="#main">Skip to main content</a>
-  <header class="usa-header usa-header-extended" role="banner" style="background-color: white;">
-  {% include site-banner.html %}
-    <div class="usa-navbar">
-      <button class="usa-menu-btn">Menu</button>
-      <div class="usa-logo" id="logo">
-        <em class="usa-logo-text">
-          <a href="{{ site.baseurl }}/" accesskey="1" title="Home">
-            <img src="{{ site.baseurl }}/images/legacylogoonwhite.png" alt="Foia.gov">
-            <h1 class="usa-header-title">FOIA.gov</h1>
-          </a>
-        </em>
-      </div>
+<a class="usa-skipnav" href="#main">Skip to main content</a>
+<header class="usa-header usa-header-extended" role="banner" style="background-color: white;">
+{% include site-banner.html %}
+  <div class="usa-navbar">
+    <button class="usa-menu-btn">Menu</button>
+    <div class="usa-logo" id="logo">
+      <em class="usa-logo-text">
+        <a href="{{ site.baseurl }}/" accesskey="1" title="Home">
+          <img src="{{ site.baseurl }}/images/legacylogoonwhite.png" alt="Foia.gov">
```

Comment:
@RyanSibley the alt text should be "FOIA.gov", right?

Reply:
nm, I think the alt text is cased like this so that screen readers will speak this as "FOIA" instead of "F-O-I-A".